### PR TITLE
business continuity 2.11 known issues

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-11766

### CODE REVIEW

File: release_notes/known_issues_continuity.adoc [hunk 1]
```diff
@@ -44,70 +44,22 @@ The `Terminating` state is a result of the Velero restore resources waiting on t
...
+//2.11:ACM-11766
 
 If the resources for the bare metal cluster are backed up and restored to a secondary hub cluster by using the {product-title-short} back up and restore feature, the managed cluster reinstalls on the nodes, which destroys the existing managed cluster. 
 
 *Note:* This only affects bare metal clusters that were deployed by using zero touch provisioning, meaning that they have `BareMetalHost` resources that manage powering on and off bare metal nodes and attaching virtual media for booting.
 
 If a `BareMetalHost` resource was not used in the deployment of the managed cluster, there is no negative impact.
 
-To work around this issue, exclude managed `BareMetalHost` resources on the primary hub cluster from performing back up and restore to the secondary hub cluster. 
+To work around this issue, ensure that the managed `BareMetalHost` resources on the primary hub cluster are no longer backed up with the managed cluster backup
```
Comment (dockerymick) [inline release_notes/known_issues_continuity.adoc]:
```suggestion
To work around this issue, ensure that the managed `BareMetalHost` resources on the primary hub cluster are no longer backed up with the managed cluster backup.
```

**Review by dockerymick — APPROVED** [review]
Added a suggestion, otherwise this looks good to me!

File: release_notes/known_issues_continuity.adoc [hunk 2]
```diff
@@ -44,70 +44,22 @@ The `Terminating` state is a result of the Velero restore resources waiting on t
 . If your `open-cluster-management-backup` namespace is already stuck in the `Terminating` state, edit all the Velero restore resources and remove the finalizers. 
 . Allow the Velero resources to delete the namespaces and resources. 
 
-[#bare-metal-reintall]
-=== Bare metal managed clusters deployed with the Infrastructure Operator by using the ZTP flow perform reinstall nodes 
-//2.9:ACM-8627
+[#bare-metal-resources]
+=== Bare metal hub resources not backed up by
```
Comment (birsanv) [inline release_notes/known_issues_continuity.adoc]: This looks incomplete .. I expect to see this content here ( from the linked jirra )

Title : 

Bare metal hub resource no longer being backed up by the managed clusters backup

Description:
If the resources for the bare metal cluster are backed up and restored to a secondary hub cluster by using the {product-title-short} back up and restore feature, the managed cluster reinstalls on the nodes, which destroys the existing managed cluster.
Note: This only affects bare metal clusters that were deployed by using zero touch provisioning, meaning that they have BareMetalHost resources that manage powering on and off bare metal nodes and attaching virtual media for booting. If a BareMetalHost resource was not used in the deployment of the managed cluster, there is no negative impact.
 

To work around this issue, the managed BareMetalHost resources on the primary hub cluster are no longer backed up with the managed cluster backup. 

 

If you have a usecase different than the above and you want the managed BareMetalHost resources on the primary hub cluster to be backed up, add the backup label to the BareMetalHost resources on the primary hub cluster: cluster.open-cluster-management.io/backup.

Read this section to learn about using this backup label to backup generic resources. https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/business_continuity/business-cont-overview#resources-that-are-backed-up

File: release_notes/known_issues_continuity.adoc [hunk 3]
```diff
@@ -44,70 +44,22 @@ The `Terminating` state is a result of the Velero restore resources waiting on t
...
+//2.11:ACM-11766
 
 If the resources for the bare metal cluster are backed up and restored to a secondary hub cluster by using the {product-title-short} back up and restore feature, the managed cluster reinstalls on the nodes, which destroys the existing managed cluster.
```
Comment (birsanv) [inline release_notes/known_issues_continuity.adoc:51]: Line 51-62 should be replaced by the section in comment above

**Review by birsanv — CHANGES_REQUESTED** [review]

File: release_notes/known_issues_continuity.adoc [hunk 4]
```diff
@@ -44,70 +44,20 @@ The `Terminating` state is a result of the Velero restore resources waiting on t
...
 . Allow the Velero resources to delete the namespaces and resources. 
 
-[#bare-metal-reintall]
-=== Bare metal managed clusters deployed with the Infrastructure Operator by using the ZTP flow perform reinstall nodes 
-//2.9:ACM-8627
+[#bare-metal-resources]
+=== Bare metal hub resource no longer backed up by the managed clusters backup
+//2.11:ACM-11766
 
 If the resources for the bare metal cluster are backed up and restored to a secondary hub cluster by using the {product-title-short} back up and restore feature, the managed cluster reinstalls on the nodes, which destroys the existing managed cluster.
```
Comment (jc-berger) [inline release_notes/known_issues_continuity.adoc:51]: We typically don't have a header, like `Description` 

If you want, I can do something like:

[#symptom-managed-clusters-destroyed]
=== Symptom: Managed clusters get destroyed 

If the resources for the bare metal cluster are backed up....

However, we typically have that structure for Troubleshooting content. But I can add it here in the Known issues as well!

File: release_notes/known_issues_continuity.adoc [hunk 5]
```diff
@@ -44,70 +44,20 @@ The `Terminating` state is a result of the Velero restore resources waiting on t
...
-apiVersion: oadp.openshift.io/v1alpha1
-blockOwnerDeletion: true
-controller: true
-kind: DataProtectionApplication
-name: dpa-sample
-uid: 52acd151-52fd-440a-a846-95a0d7368ff7
-----
-
-. Change the value of `spec.deployments.label.spec.replicas` back to `1` to start the data protection application process with the new settings. 
+To learn more about using this backup label to backup generic resources, see the topic, link:../business_continuity/backup_restore/backup_arch.adoc#resources-that-are-backed-up[Cleaning the hub cluster after restore].
```
Comment (birsanv) [inline release_notes/known_issues_continuity.adoc]: The name of the link is incorrect ( Cleaning the hub cluster after restore ) 

It should be : Resources that are backed up

**Review by birsanv — CHANGES_REQUESTED** [review]
just one small update, everything else looks good

**Review by birsanv — APPROVED** [review]
lgtm

**Pull request merged**